e2e-tests/cypress/e2e/journeys/insurance/policy-and-exports/change-your-answers/change-your-answers-single-contract-policy.spec.js

```diff
@@ -0,0 +1,262 @@
+import currencies from '../../../../../fixtures/currencies';
+
+const {
+  POLICY_AND_EXPORTS: {
+    CHECK_YOUR_ANSWERS,
+    SINGLE_CONTRACT_POLICY_CHANGE,
+  },
+} = ROUTES.INSURANCE;
+
+const {
+  INSURANCE: {
+    POLICY_AND_EXPORTS: {
+      CONTRACT_POLICY: {
+        REQUESTED_START_DATE,
+        CREDIT_PERIOD_WITH_BUYER,
+        POLICY_CURRENCY_CODE,
+        SINGLE: { CONTRACT_COMPLETION_DATE, TOTAL_CONTRACT_VALUE },
+      },
+    },
+  },
+} = FIELD_IDS;
+
+const { taskList } = partials.insurancePartials;
+
+const task = taskList.prepareApplication.tasks.policyTypeAndExports;
+
+const { summaryList } = checkYourAnswersPage;
+
+const assertChangePageUrl = (referenceNumber, fieldId) => {
+  const expected = `${Cypress.config('baseUrl')}${INSURANCE_ROOT}/${referenceNumber}${SINGLE_CONTRACT_POLICY_CHANGE}#${fieldId}-label`;
+  cy.url().should('eq', expected);
+};
+
+const assertAnswersPageUrl = (referenceNumber, fieldId) => {
+  const expected = `${Cypress.config('baseUrl')}${INSURANCE_ROOT}/${referenceNumber}${CHECK_YOUR_ANSWERS}#${fieldId}-label`;
+
+  cy.url().should('eq', expected);
+};
+
+const assertSummaryListRowValue = (fieldId, expected) => {
+  checkText(
+    summaryList[fieldId].value(),
+    expected,
+  );
+};
+
+context('Insurance - Policy and exports - Check your answers - Single contract policy - As an exporter, I want to change my answers to the type of policy and exports section', () => {
+  let referenceNumber;
+
+  before(() => {
+    cy.visit(ROUTES.INSURANCE.START, {
+      auth: {
+        username: Cypress.config('basicAuthKey'),
+        password: Cypress.config('basicAuthSecret'),
+      },
+    });
+
+    cy.submitInsuranceEligibilityAndStartApplication();
+
+    task.link().click();
+
+    cy.completeAndSubmitPolicyTypeForm(FIELD_VALUES.POLICY_TYPE.SINGLE);
+    cy.completeAndSubmitSingleContractPolicyForm();
+    cy.completeAndSubmitAboutGoodsOrServicesForm();
+
+    getReferenceNumber().then((id) => {
+      referenceNumber = id;
+
+      const expected = `${Cypress.config('baseUrl')}${INSURANCE_ROOT}/${referenceNumber}${CHECK_YOUR_ANSWERS}`;
+      cy.url().should('eq', expected);
+    });
+  });
+
+  beforeEach(() => {
+    Cypress.Cookies.preserveOnce('_csrf');
+    Cypress.Cookies.preserveOnce('connect.sid');
+  });
+
+  describe('single policy type answers', () => {
+    describe(REQUESTED_START_DATE, () => {
+      const fieldId = REQUESTED_START_DATE;
+
+      describe('when clicking the `change` link', () => {
+        it(`should redirect to ${SINGLE_CONTRACT_POLICY_CHANGE}`, () => {
+          summaryList[fieldId].changeLink().click();
+
+          assertChangePageUrl(referenceNumber, fieldId);
+        });
+      });
+
+      describe('form submission with a new answer', () => {
+        const newAnswer = {
+          ...application.POLICY_AND_EXPORTS[fieldId],
+          day: application.POLICY_AND_EXPORTS[fieldId].day + 1,
+          month: application.POLICY_AND_EXPORTS[fieldId].month + 1,
```

Comment:
will this not error when this test runs in december? or date is 31st? same with below